src/styles/main.scss

```diff
@@ -1,3 +1,54 @@
+@import 'header';
+@import 'card';
+@import 'cards';
+@import 'variables';
+
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+}
+
 body {
   margin: 0;
+  display: block;
+}
+
+.cards {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  gap: 46px 48px;
+  justify-content: center;
```

Comment:
The justify-content: center; and align-items: center; properties are not needed in grid layout and will not have any effect. These properties are typically used in a flex container.